accounts/abi/abi_extra_test.go

```diff
@@ -0,0 +1,45 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package abi
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// Note: This file contains tests in addition to those found in go-ethereum.
+
+const TEST_ABI = `[ { "type" : "function", "name" : "receive", "inputs" : [ { "name" : "memo", "type" : "bytes" }], "outputs":[{"internalType":"bool","name":"isAllowed","type":"bool"}] } ]`
+
+func TestUnpackInputIntoInterface(t *testing.T) {
+	abi, err := JSON(strings.NewReader(TEST_ABI))
+	require.NoError(t, err)
+
+	input := []byte("hello")
+	data, err := abi.Pack("receive", input)
+	require.NoError(t, err)
+
+	var v []byte
```

Comment:
Can we add a test case for an interface in the same style as for `UnpackIntoInterface` ?https://github.com/ava-labs/subnet-evm/blob/master/accounts/abi/abi_test.go#L775